static/js/common.js

```diff
@@ -58,8 +58,25 @@ function password_quality(password, bar, password_field) {
     return acceptable;
 }
 
+function password_warning(password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
```

Comment:
@rishig Yeah makes sense. But it should be 6 right? That's the default value in settings.py.